source/_docs/configuration.markdown

```diff
@@ -40,6 +40,6 @@ If you can't see your integration listed there, you will need to restart Home As
 
 ## Migrating to a new system
 
-The preferred way of migrating to a new system is by {% my supervisor_backups title="making a backup" %}.
+The preferred way of migrating to a new system is by {% my supervisor_backups title="making a backup" %}. Once you have created the backup on the old system, you can download it to the system that is running the Home Assistant frontend. From there you can upload it to your new system using the *Upload backup* menu option of the *Backups* menu. A restore of the uploaded backup on the new system concludes the migration.
```

Comment:
A restore of backups is offered when setting up / onboarding the new system. This path is not the "normal" path.

Reply:
Then I would argue that a statement similar to "A restore of backups is offered when setting up / onboarding the new system" should be added to the documentation. I was preparing a migration myself and found the documentation on how to use the backup of the old system to be missing. Actually, the documentation only indicates that a backup should be made; nothing on what to do with the backup.
If you like I can add the statement to the documentation.

Reply:
have a look at #26530,,, (my simplified 'Commits on Mar 31, 2023') it's adressing the same topic :)

I'm new to PR's and it seems I've c&%¤ed something up,,, 
#learningcurve :)

Reply:
I think I have addressed the requested changes.